README.md

```diff
@@ -63,7 +63,7 @@ $ ouroboros --interval 300 --loglevel debug
 ## Examples
 
 ### Monitor for updates for latest tag
- Instead of updating to your original image tag you can specify if you would like Ouroboros to update all containers to `latest`.  
+ Instead of updating to your original image tag you can specify if you would like Ouroboros to update all containers to `latest`.
```

Comment:
gotta add the double space back. messes up formatting

Reply:
Good catch.